lnwallet/wallet.go

```diff
@@ -99,6 +99,10 @@ type InitFundingReserveMsg struct {
 	// to this channel.
 	RemoteFundingAmt btcutil.Amount
 
+	// FundUpToMaxAmt defines if channel funding should try to add as many
+	// funds to LocalAmt as possible until FundMaxAmt is reached.
+	FundUpToMaxAmt btcutil.Amount
```

Comment:
Because of the check in the `wallet_assembler`, the `LocalAmt` becomes the minimum amount of coins that need to be available if this is set. Maybe this should be mentioned here.

Reply:
I have tried to clarify this with the new commit.